src/Copy/BrandCopy.elm

```diff
@@ -447,88 +447,88 @@ brandCopy key =
 
         StoriesTitleH2 ->
-            CopyText "✨"
+            CopyText "Stories"
 
         StoriesTeaserMoreLink title ->
-            CopyText ("✨ " ++ title ++ "✨")
+            CopyText ("Read " ++ title ++ "'s Story")
 
         StoryCardH3 cardId ->
-            CopyText ("✨ " ++ String.fromInt cardId ++ " ✨")
+            CopyText ("Part " ++ String.fromInt cardId ++ " of 4")
 
         StoryCardStartJourneyPrompt ->
-            CopyText "✨"
+            CopyText "Everyone's story is different, but you will find someone who can help you. Start your journey now."
 
         StoryNotFoundTitle ->
-            CopyText "✨"
+            CopyText "Coming soon"
 
         StoryNotFoundTeaser ->
-            CopyText "✨"
+            CopyText "We're working on it"
 
         StoryNotFoundQuote ->
-            CopyText "✨"
+            CopyText "Story coming soon."
 
         StoryNotFoundImageAlt ->
-            CopyText "✨"
+            CopyText "Placeholder image"
 
         StoryNotFoundMessage ->
-            CopyText "✨"
+            CopyText "Check back soon"
 
         StoryOneTitle ->
-            CopyText "✨"
+            CopyText "Diane"
 
         StoryOneTeaser ->
-            CopyText "✨"
+            CopyText "Diane felt her complaint about her husband’s treatment was badly handled, but a weight was lifted when she found an advocate."
 
         StoryOneTeaserImageAlt ->
             CopyText "✨"
 
         StoryOne1Quote ->
-            CopyText "✨"
+            CopyText "When my husband was first diagnosed with cancer, the treatment was amazing, he was in chemotherapy within 36hours. But later, an A&E visit went very badly when protocols and procedures were ignored. We put in a complaint to the Health Care Trust, and one sentence in their reply made us so angry."
 
         StoryOne1ImageAlt ->
             CopyText "✨"
 
         StoryOne1Message ->
-            CopyText ""
+            CopyText "They said my husband’s treatment had been appropriate but we knew the protocol for neutropenic sepsis."
 
         StoryOne2Quote ->
-            CopyText "✨"
+            CopyText "The leaflet about an advocate came with the reply. I had spoken to my husband. “What do you reckon, it’s a charity, we don’t have to but shall we give it a go?”. There were lots of forms but it was all very smooth. It was actually the week of my husband’s funeral when the worker called to arrange the first meeting, but I thought “I really do want to get on with this”."
```

Comment:
Are the double special chars rendering ok everywhere?

Reply:
no issue on Chrome and Firefox for linux (fedora 30), latest versions. I guess if anything, it will be uncovered during testing?

Reply:
Make an issue to look at that specifically.

Reply:
a separate issue or an AC item on the issue for testing? at the moment, it looks good on Chrome and Firefox but for sure it needs proper testing on other broswers